### PR TITLE
RSA10g.

### PR DESCRIPTION
**Pull request by EvgenyKarkan**

### CODE REVIEW

File: Spec/Auth.swift [hunk 1]
```diff
@@ -1582,6 +1582,7 @@ class Auth : QuickSpec {
                     authOptions.authParams?.append(NSURLQueryItem(name: "body", value: token))
                     authOptions.authHeaders = ["X-Ably":"Test"]
                     authOptions.force = true
+                    authOptions.queryTime = true
```
Comment (mattheworiordan) [inline Spec/Auth.swift]: You've added `queryTime` in, yet there is no check to assert whether the time has been requested from Ably. We need to check this else I don't see the point of this.

Comment (EvgenyKarkan) [inline Spec/Auth.swift]: Got it, I'll add an assertion to be sure that `time` function has been called.

Comment (mattheworiordan) [inline Spec/Auth.swift]: Thanks

**Comment by EvgenyKarkan** [comment]
Please review and merge if everything looks good

**Comment by ricardopereira** [comment]
Will change the base to `master`.

**Comment by ricardopereira** [comment]
Squashed.

**Pull request merged**